docs/operations/metrics.md

```diff
@@ -342,6 +342,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`killTask/availableSlot/count`| Number of available task slots that can be used for auto kill tasks in the auto kill run. This is the max number of task slots minus any currently running auto kill tasks.                                                                                                                                                                                                                                                                                                                     | |Varies|
 |`killTask/maxSlot/count`| Maximum number of task slots available for auto kill tasks in the auto kill run.                                                                                                                                                                                                                                                                                                                                                                                                                | |Varies|
 |`kill/task/count`| Number of tasks issued in the auto kill run.                                                                                                                                                                                                                                                                                                                                                                                                                                                    | |Varies|
+|`kill/candidateUnusedSegments/count`|Number of candidate unused segments to be deleted from the metadata store in an auto kill run.|`dataSource`|Varies|
```

Comment:
Reading at this doc and the name of the metric, it's not very clear what it represents.
Is it the number of segments that the coordinator _decided to kill_ in a datasource or the number of segments that _should be killed_ in a datasource? Also, should it have `interval` as a dimension?

Reply:
> Is it the number of segments that the coordinator decided to kill in a datasource or the number of segments that should be killed in a datasource? 

It's the latter - clarified the docs a wee bit. Let me know if you have a better suggestion. I called it candidate because it's ultimately the underlying kill task that can say this for sure.

>Also, should it have interval as a dimension?

No, there can be multiple intervals.

Reply:
> No, there can be multiple intervals, so an interval dimension can explode the metric.

Makes sense.